BLSQ/iaso
# IA-2315:User roles api should be available in read only mode

Explain what problem this PR is resolving
- When trying to access user when user role permission is not available it generates some errors
Related JIRA tickets : [IA-2315](https://bluesquare.atlassian.net/browse/IA-2315)

## Self proofreading checklist


## Changes
- Let every user read(GET) a user role

## How to test
- Create a new user without the user role permission
- Connect with the created user
- Go to the user list
- Try to edit an existing user, there should be no error
## Print screen / video

![Peek 2023-07-17 10-39](https://github.com/BLSQ/iaso/assets/19631540/df71e510-fee1-4ec2-8178-adc2061a4280)

![Peek 2023-07-17 10-42](https://github.com/BLSQ/iaso/assets/19631540/8db699ff-8da7-4210-97b6-11bd3615d7e7)


[IA-2315]: https://bluesquare.atlassian.net/browse/IA-2315?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
